charts/exposurelog/templates/deployment.yaml

```diff
@@ -39,15 +39,15 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           ports:
             - name: http
-              containerPort: 8080
+              containerPort: 80
```

Comment:
I think you want `{{ .Values.service.port }}` here as well. The exposure app will need to be listening on this port also,  so if the port changes you can set it in the `values.yaml` and you don't need to change the chart.

Reply:
Good suggestion. Done.